app/lib/screens/history/history.dart

```diff
@@ -1,6 +1,16 @@
 import 'package:flutter/material.dart';
 
-class TravelHistory extends StatelessWidget {
+class TravelHistory extends StatefulWidget {
+  @override
+  _TravelHistoryState createState() => _TravelHistoryState();
+}
+
+class _TravelHistoryState extends State<TravelHistory> {
```

Comment:
Could you tell me what the differences between `state` `page` and `widget` are to you?

Reply:
Widget is the description of the UI (e.g. Padding., etc). Two categories of widgets, Stateless and Stateful. 
Stateless - UI/data that will never change.
Stateful - UI/data that can change.
I know in the future, the UI in travel history will change (e.g. adding travel).
Depending on your business logic, you need to decide whether a widget should be stateful or not. 

I encourage everyone (@holtzmak , @mansiPaTeLL , @SanaKhan1997 ) to look into different states of flutter and how to manage it.  Here's where to [start](https://flutter.dev/docs/development/data-and-backend/state-mgmt).